src/dut.rs

```diff
@@ -123,12 +169,20 @@ impl MonitoredDut {
             match child.try_status()? {
                 None => {
                     self.reconnecting = false;
-                    Ok(format!(
+                    let mut status = format!(
                         "{:<31}\t127.0.0.1:{:<5}\t{}",
                         &self.dut,
                         self.port,
                         &self.ssh.host_and_port()
-                    ))
+                    );
+                    self.fwports.iter().for_each(|fwp: &PortForwarding| {
```

Comment:
I'm not really a fan of this for_each stuff when you could just map the whole vec, produce a string for each element, then join the string using vec/iter's methods

Reply:
Hm, OK. I just feel that loops on the list twice, but it maybe C programmer's thought.